packages/keystone/databaseAdapters/ReplicaKnexAdapter.js

```diff
@@ -0,0 +1,115 @@
+const { KnexAdapter } = require('@keystonejs/adapter-knex')
+const { knex } = require('knex')
+const { get, omit } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+
+const logger = getLogger('replicaKnexAdapter')
+
+class ReplicaKnexAdapter extends KnexAdapter {
+    constructor (props) {
+        super(omit(props, ['connection']))
+        this.readConnection = get(props, ['connection', 'read'])
+        this.writeConnection = get(props, ['connection', 'write'])
+    }
+
+    async _connect () {
+        this.knex = knex({
+            client: 'postgres',
+            pool: { min: 0, max: 1 },
+            connection: this.writeConnection,
+        })
+
+        this.knexMaster = knex({
```

Comment:
naming is a bit strange to me. It's usually master / replica or write / read, here is a combo :)

Reply:
Ok, I'll refactor this later